.github/workflows/markdown-lint.yml

```
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches: [ main, master, draft, draft* ]
+  pull_request:
+    branches: [ main, master ]
+
+permissions: {}  # Setting default permissions to none for enhanced security
+
+jobs:
+  markdown-lint:
+  permissions: read
+    runs-on: ubuntu-latest
```

actionlint

could not parse as YAML: yaml: line 15: mapping values are not allowed in this context

(syntax-check)